README.md

```diff
@@ -211,13 +220,13 @@ Putting it all together, we can run the `hello` eval with the `manual`
 tool as follows:
 
 ```sh
-./gradbench run --eval "uv run python/gradbench/gradbench/evals/hello/run.py" --tool "python3 python/gradbench/gradbench/cpp.py manual"
+./gradbench repo run --eval "$ uv run python/gradbench/gradbench/evals/hello/run.py" --tool "$ python3 python/gradbench/gradbench/cpp.py manual"
 ```
 
 Or without using uv:
 
 ```sh
-PYTHONPATH=python/gradbench/:$PYTHONPATH ./gradbench run --eval "python3 python/gradbench/gradbench/evals/hello/run.py" --tool "python3 python/gradbench/gradbench/cpp.py manual"
+PYTHONPATH=python/gradbench/:$PYTHONPATH ./gradbench repo run --eval "$ python3 python/gradbench/gradbench/evals/hello/run.py" --tool "$ python3 python/gradbench/gradbench/cpp.py manual"
```

Comment:
This should still be `run`, shouldn't it? And those dollar signs must certainly be superfluous.

Reply:
Oh wait, no, I see.